packages/elements/test/component-factory-strategy_spec.ts

```diff
@@ -239,6 +255,39 @@ describe('ComponentFactoryNgElementStrategy', () => {
          expect(componentRef.destroy).toHaveBeenCalledTimes(1);
        }));
   });
+
+  describe('runInZone', () => {
+    it('should have zones (baseline)', () => {
+      expect(strategy['zoneless']).toEqual(false);
+      expect(strategy['ngZone']).toBeTruthy();
+      expect(strategy['elementZone']).toBeTruthy();
+    });
+
+    it('should run already in original zone', () => {
```

Comment:
Consider something more descriptive, such as:

```suggestion
    it('should run the callback directly when invoked in element\'s zone', () => {
```